website/sidebars.js

```diff
@@ -33,100 +33,136 @@ module.exports = {
     },
     {
       type: "category",
-      label: "Go SDK",
+      label: "Dagger Cloud",
       collapsible: true,
       collapsed: true,
       items: [
         {
           type: "doc",
           label: "Overview",
-          id: "current/sdk/go/index",
+          id: "current/cloud/index",
         },
-        "current/sdk/go/install",
         {
           type: "doc",
           label: "Get Started",
-          id: "current/sdk/go/get-started",
+          id: "current/cloud/get-started",
         },
         {
-          type: "link",
-          label: "Reference 🔗",
-          href: "https://pkg.go.dev/dagger.io/dagger",
-        },
-      ],
-    },
-    {
-      type: "category",
-      label: "Node.js SDK",
-      collapsible: true,
-      collapsed: true,
-      items: [
-        {
-          type: "doc",
-          label: "Overview",
-          id: "current/sdk/nodejs/index",
-        },
-        "current/sdk/nodejs/install",
-        {
-          type: "doc",
-          label: "Get Started",
-          id: "current/sdk/nodejs/get-started",
-        },
-        {
-          type: "doc",
+          type: "category",
           label: "Reference",
```

Comment:
There is an error in the navigation, wherein some of these nodes do not unroll correctly. Can @slumbering or @crjm please have a look at this?

Reply:
I tried a few times and cannot reproduce.
What browser are you using?

Reply:
This seems to be an issue with my local env. Neither @d3rp3tt3 nor @slumbering are able to reproduce it either.